docs/pages/webhook/events/AUCTION_BID_CREATED.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'AUCTION_BID_CREATED'
+---
+
+# AUCTION_BID_CREATED
+
+This event occurs whenever a bid on an auction is created.
+
+## Event
+
+```ts
+type AUCTION_BID_CREATED = {
```

Comment:
The type custom type are not explain at all, could just put their defition here also:
```suggestion
type Uint256 = string;
type IsoDate = string;

type AUCTION_BID_CREATED = {
```